e2e-tests/commands/insurance/complete-and-submit-about-goods-or-services-form.js

```diff
@@ -16,7 +16,8 @@ const {
 } = FIELD_IDS;
 
 export default () => {
-  cy.keyboardInput(aboutGoodsOrServicesPage[DESCRIPTION].input(), application.EXPORT_CONTRACT[DESCRIPTION]);
+  cy.keyboardInput(aboutGoodsOrServicesPage[DESCRIPTION].textarea(), application.EXPORT_CONTRACT[DESCRIPTION]);
```

Comment:
any reaon why this is not field().textArea and line 21 is not field().input?

Reply:
So with these changes, the text areas and country inputs are the only fields that are different when using the generic `field` helper, needs some extra work, would like to clean up after.

Basically the country inputs have a single ID e.g `country`, instead of `fieldId-input` like all other fields. Same with textareas, they have a single ID due to GOV design limitation.

Want to see what else we can do to clean these up but don't want to bloat this PR anymore 🙃